### PR TITLE
fix window.setEventHandler, fix all touch events

### PR DESCRIPTION
**Pull request by Bloblblobl**
@zgalant

fixes the setEventHandler functions to clean up event handlers properly when they are changed, updates it to allow setting handlers on specific elements instead of just the window, and then updates all touch events to use setEventHandler

### CODE REVIEW

**Comment by Bloblblobl** [comment]
tested this out locally by emulating the touch events so at least it shouldn't break anything, @zgalant let me know if it works when you get the chance to try it

**Comment by zgalant** [comment]
https://share.icloud.com/photos/084EdpSNI5Yao9rR2AtK4C-5g
Works!

File: src/entry/app.js [hunk 1]
```diff
@@ -18,15 +18,44 @@ import {
     inappPrivacyPolicy,
 } from "./inapp";
 
-window.eventHandlers = {};
-window.setEventHandler = function (event, handler) {
-    const existingHandler = window.eventHandlers[event];
-    if (existingHandler === undefined) {
-        window.removeEventListener(event, existingHandler);
+
```
Comment (zgalant) [inline src/entry/app.js:21]: helpful comment!

**Comment by Bloblblobl** [comment]
> Works!
sweet 😄

**Pull request merged**